src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+                preFlopOutPositionPlayerGotRaise(player1, player2);
+            } else if (smallBlindCallPreFlop && gameState.getGamePhase() == GamePhase.PREFLOP) {
+                System.out.println(player2.getNamePlayer() + " raise to " + raise + "$");
+                upadePotStackAmountInHandSmallBlindPlayer(raise - bigBlind, player2);
+                smallBlindCallPreFlop = false;
+                player2.increaseStackPlyer(maxValueRaiseSmallBlind);
+                player2.decreaseAmountInHand(maxValueRaiseSmallBlind);
+                gameState.decreasePot(maxValueRaiseSmallBlind);
+                maxValueRaiseSmallBlind = raise;
+            } else {
+                if (raise > maxValueRaiseSmallBlind) {
+                    player2.increaseStackPlyer(maxValueRaiseSmallBlind);
+                    player2.decreaseAmountInHand(maxValueRaiseSmallBlind);
+                    gameState.decreasePot(maxValueRaiseSmallBlind);
+                    maxValueRaiseSmallBlind = raise;
+                }
+                System.out.println(player2.getNamePlayer() + " raise to " + raise + "$");
+                upadePotStackAmountInHandSmallBlindPlayer(raise, player2);
+            }
+        }
+    }
+
+    public void makeBetBigBlingPlayer(Player player1, Player player2) {
+        double bet = 0;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Podaj size beta");
+        bet = scanner.nextDouble();
+        if (bet >= player1.stack) {
+            System.out.println(player1.getNamePlayer() + " All in " + player1.getStackPlayer() + "$");
+            bet = player1.stack;
+            upadePotStackAmountInHandBigBlindPlayer(bet, player1);
+            preFlopInPositionPlayerCantCheck(player1, player2);
+        } else {
+            System.out.println(player1.getNamePlayer() + " bets " + bet + "$");
+            upadePotStackAmountInHandBigBlindPlayer(bet, player1);
+        }
+    }
+
+    public void makeBoardDependsGamePhase() {
+        if (gameState.getGamePhase() == GamePhase.FLOP) {
+            layCards(gameState);
+            gameState.setGamePhase(GamePhase.TURN);
+        } else if (gameState.getGamePhase() == GamePhase.TURN) {
+            layCards(gameState);
+            gameState.setGamePhase(GamePhase.RIVER);
+        } else {
+            layCards(gameState);
+        }
+    }
+
+    public void resetMaxValueRaises() {
+        maxValueRaiseBigBlind = 0;
+        maxValueRaiseSmallBlind = 0;
+    }
+
+    public void upadePotStackAmountInHandBigBlindPlayer(double amount, Player player1) {
+        gameState.updatePot(amount);
+        player1.updateStackPlayer(amount);
+        player1.updateAmountInHand(amount);
+    }
+
+    public void upadePotStackAmountInHandSmallBlindPlayer(double betOrRaise, Player player2) {
+        gameState.updatePot(betOrRaise);
+        player2.updateStackPlayer(betOrRaise);
+        player2.updateAmountInHand(betOrRaise);
+    }
```

Comment:
te 2 funkcje robia to samo wiec bez sensu jest je rozbijac na 2 tylko po to zeby nazwac to BigBlind/SmallBlind, powinna byc jedna funkcja np updatePotAndPlayerAmount(double amount, Player player){
        gameState.updatePot(amount);
        player.updateStackPlayer(amount);
        player.updateAmountInHand(amount);
}

Reply:
poprawiles na upadePotStackAmountInHandPlayer() nazwa troche slaba bo malo przejrzyste co updejtujesz zmien na updatePotAndStackAmount